pkg/apis/common/common.go

```diff
@@ -132,6 +132,8 @@ type TLSConfig struct {
 	ClientCertSecret *corev1.SecretKeySelector `json:"clientCertSecret,omitempty" protobuf:"bytes,2,opt,name=clientCertSecret"`
 	// ClientKeySecret refers to the secret that contains the client key
 	ClientKeySecret *corev1.SecretKeySelector `json:"clientKeySecret,omitempty" protobuf:"bytes,3,opt,name=clientKeySecret"`
+	// If true, skips creation of TLSConfig with certs and creates an empty TLSConfig. Optional default is false
+	InsecureSkipVerify bool `json:"insecureSkipVerify,omitempty" protobuf:"varint,4,opt,name=insecureSkipVerify"`
```

Comment:
The comments format matter.
```
// xxxxx description goes here. xxxxxx
// +optional
InsecureSkipVerify bool `json:"insecureSkipVerify,omitempty" protobuf:"varint,4,opt,name=insecureSkipVerify"`
```

Reply:
changed and added the code-gen files. The tests are passing locally.